avocado/utils/iso9660.py

```diff
@@ -160,7 +166,30 @@ def close(self):
         pass
 
 
-class Iso9660IsoInfo(BaseIso9660):
+class BaseIso9660MntDirMount(BaseIso9660):
```

Comment:
One other way to do this (up to your style of choosing though) is to a "mix in" class.  I think it would be easier to follow that this adds `mnt_dir` support "into the mix" of some (most, really) iso classes (except `Iso99660Mount` itself).

Apart from the style suggestion, this looks good.